### ISSUE TITLE
creating 1km grid file

### ISSUE THREAD

**Issue by amyjsdavis** [trias-project/eea-ref-grid-1km#1]
I used a different procedure for creating the 1km grid. I had one base grid for most of Europe and stitched it together with the individual 1km EEA grids for Norway, Montenegro and Serbia which were not on the base grid. Do you me to edit eea-ref-grid-1km to reflect the procedure I used to create the grid? Or I will happy to create a separate repository to document my steps.

**Comment by peterdesmet** [comment]
Thanks @amyjsdavis. The first thing I want to figure out - and I already mentioned this to @damianooldoni - is if we even need this repo. We don't need a grid for the occ-processing, because we calculate the grid cells ourselves.

If the output of the occ-processing is sufficient to start your analysis, I wonder if we need to document how to create a eea grid **geospatial** file.

**Comment by amyjsdavis** [comment]
@peterdesmet : I agree, it is probably not needed.

**Comment by damianooldoni** [comment]
Indeed, for the data cube we found out a fast work-around to avoid very long and memory consuming spatial joins. See: https://github.com/trias-project/occ-processing/blob/master/src/europe/assign_grid.Rmd#L408-L418.
The cell code an occurrence belongs to is easily found by using `X` and `Y` in EPSG 3035.
generally speaking, this method relies on a feature which is common to all scales of EEA grids, i.e. 1, 10 or 100 km. See in my Gist a general function to find the EEA cell code of a data frame of coordinates:
https://gist.github.com/damianooldoni/bdc0caa3edd5609fb6746eb7eff36099
@amyjsdavis : up to now it is not needed to be part of our `trias` package, but let me know if you ever need it.

The GBIF occurrence data I downloaded for the European data cube fall within a simple polygon drawn around Europe: see https://github.com/trias-project/occ-processing/blob/master/src/europe/download.Rmd#L59-L66. So, it could be that some cells in the cube are not official cell codes of what EEA coniders as Europe.
